<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modeManuelWidget</class>
 <widget class="QWidget" name="modeManuelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VL_menu">
    <item>
     <layout class="QVBoxLayout" name="VL_menuTransaction">
      <item>
       <widget class="QLabel" name="pairelabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10777215</height>
         </size>
        </property>
        <property name="text">
         <string>Paire Devise</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL_infoDate">
        <item>
         <layout class="QGridLayout" name="gridLayout_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="base">
          <property name="text">
           <string>Devise Base</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_montantBase">
          <property name="text">
           <string>MontantB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="contrepartie">
          <property name="text">
           <string>Contre partie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_montantContrepartie">
          <property name="text">
           <string>MontantC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Interet Broker</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_broker">
          <property name="text">
           <string>Broker</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_date">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="openPrice">
          <property name="text">
           <string>Open price</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="montant_Edit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL_achatvente">
        <item>
         <widget class="QPushButton" name="pushButton_achat">
          <property name="text">
           <string>Achat</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/TradingSimulator/evolutionCours/icons/buy.png</normaloff>:/TradingSimulator/evolutionCours/icons/buy.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_vente">
          <property name="text">
           <string>Vente</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/TradingSimulator/evolutionCours/icons/sell.png</normaloff>:/TradingSimulator/evolutionCours/icons/sell.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_annulerTransaction">
        <property name="text">
         <string>Annuler</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/TradingSimulator/evolutionCours/icons/undo.png</normaloff>:/TradingSimulator/evolutionCours/icons/undo.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
